@navikt/core/icons/meta.json

```diff
@@ -509,12 +509,24 @@
     "created_at": "2021-01-11T13:37:55.678Z",
     "pageName": "Communication"
   },
+  {
+    "name": "DialogFilled",
+    "description": "",
+    "created_at": "2022-03-30T07:30:17.398Z",
+    "pageName": "People"
+  },
   {
     "name": "DialogFilled",
     "description": "",
     "created_at": "2021-01-11T13:37:55.613Z",
     "pageName": "Communication"
   },
```

Comment:
Vi kan vel ikke ha to ikoner som heter det samme? Vi har `Dialog filled` på siden People, `dialog-filled` på siden Communication. `dialog` på Communication og `Dialog outline` på People. 🤔